tutorials/scripting/gdextension/gdextension_cpp_example.rst

```diff
@@ -0,0 +1,643 @@
+.. _doc_gdextension_cpp_example:
+
+GDExtension C++ example
+====================
+
+Introduction
+------------
+
+The C++ bindings for GDExtension are built on top of the C GDExtension API
+and provide a nicer way to "extend" nodes in Godot using C++. This new system allows the 
+extension of Godot to nearly the same level as statically linked C++ modules.
+
+You can download the included example in the test folder of the godot-cpp repository `on
+GitHub <https://github.com/godotengine/godot-cpp>`__.
+
+Setting up the project
+----------------------
+
+There are a few prerequisites you'll need:
+
+- a Godot 4.x executable,
+- a C++ compiler,
+- SCons as a build tool,
+- a copy of the `godot-cpp
+  repository <https://github.com/godotengine/godot-cpp>`__.
+
+See also :ref:`Compiling <toc-devel-compiling>` as the build tools are identical
+to the ones you need to compile Godot from source.
+
+You can download these repositories from GitHub or let Git do the work for you.
+Note that these repositories have different branches for different versions
+of Godot. GDExtension modules written will not work in older versions of Godot (3.x) 
+and vice versa, so make sure you download the correct branch.
```

Comment:
This paragraph refers to "repositories", plural, whereas the instructions immediately above only talk about a single repository.